borb/pdf/canvas/layout/image/image.py

```diff
@@ -73,10 +73,10 @@ def __init__(
             border_width=border_width,
             font_size=Decimal(12),
             horizontal_alignment=horizontal_alignment,
-            margin_bottom=margin_bottom or Decimal(5),
-            margin_left=margin_left or Decimal(5),
-            margin_right=margin_right or Decimal(5),
-            margin_top=margin_top or Decimal(5),
+            margin_bottom=margin_bottom if margin_bottom is not None else Decimal(5),
```

Comment:
Please add a test to prove the new behaviour works.
Also, please provide me with a signed copy of the contributors license agreement.